backend/models/bill.js

```diff
@@ -1,6 +1,11 @@
 const mongoose = require('mongoose');
 
 const billSchema = new mongoose.Schema({
+  userId: {
```

Comment:
@jerichobetton has already implemented (at my suggestion) adding `username` to the bill model. It doesn't matter which you use, but will need to decide together before you merge this as it will cause a conflict in `bill.js`